engine/core/IgeEngine.js

```diff
@@ -2335,7 +2355,7 @@ var IgeEngine = IgeEntity.extend({
 
 		return item;
 	},
-	isMobileDevice: function () {
+	isMobile: (function () {  // cache value
```

Comment:
Could you explain to me why we immediately invoke this?

Reply:
it used to be invoked and cached here: https://github.com/moddio/taro/pull/300/files#diff-bbc42b4b2e957c22446e44954bb292b71d6f6111f72a79d1824ad9978ce894c9L13

the function always returns the same value because it depends on user agent which does not change, so it is enough to be invoked once on the fly, cached as boolean, and than later queried when needed.

Reply:
cool. thanks for the explanation 👍